index.js

```diff
@@ -2,6 +2,9 @@ import express from 'express';
 import dotenv from 'dotenv';
 dotenv.config();
 import mongoose from 'mongoose';
+import checkAuth from './utils/checkAuth.js'
+import { loginValidation, registerValidation } from './validations/auth.js';
+import { register, login, getMe } from './controllers/UserController.js';
```

Comment:
Make a general import so that it is clear to which controller this function belongs:
```suggestion
import * as UserController from './controllers/UserController.js';
```